src/utilities/util_dump.c

```diff
@@ -193,8 +203,23 @@ util_dump(WT_SESSION *session, int argc, char *argv[])
         if (dump_config(session, simpleuri, cursor, pretty, hex, json) != 0)
             goto err;
 
+        if (start_key != NULL) {
+            cursor->set_key(cursor, start_key);
```

Comment:
Nice use of bounded cursors!

Reply:
@luke-pearson, I did not see any issue so far but should I reset the cursor since I have set a key to set the bounds? Is it dangerous to call `next/prev` on the cursor after setting the bounds?

Reply:
No the intent of bounds is that you do call cursor->next(), cursor->prev() after setting them. It will start at the beginning or end of the range depending on which call you make. If you call cursor->reset() your bounds will be removed.